Sources/CoreMetrics/Metrics.swift

```diff
@@ -172,8 +172,34 @@ public class Gauge: Recorder {
     }
 }
 
-public enum TimeUnit {
-    case nanoseconds, milliseconds, seconds, minutes, hours, days
+public struct TimeUnit: Equatable {
+    private enum Code: Equatable {
+        case nanoseconds
+        case microseconds
+        case milliseconds
+        case seconds
+        case minutes
+        case hours
+        case days
+    }
+
+    private let code: Code
+    let scaleFromNanoseconds: UInt64
```

Comment:
Should this be `public`? Otherwise no backing metrics module can use it and would need to know the names.

Reply:
Also good call, `Double` would be weird with `Equatable`.